sdk/identity/ci.yml

```diff
@@ -73,12 +73,14 @@ extends:
     LiveTestStages:
       - template: /eng/pipelines/templates/stages/archetype-sdk-tests-isolated.yml
         parameters:
+          PersistOidcToken: true
+          Location: 'westus2'
           CalledFromClient: true
           ServiceDirectory: identity
           SupportedClouds: 'Public,UsGov,China'
-          UseFederatedAuth: false
-          EnvVars:
-            SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+#          EnvVars:
+#            AZ_TENANT_ID: $(IDENTITY_TENANT_ID)
+#            AZ_CLIENT_ID: $(IDENTITY_CLIENT_ID)
           CloudConfig:
             Public:
               SubscriptionConfigurations:
```

Comment:
You should remove `$( sub-config-azure-cloud-test-resourced)` and investigate if `$(sub-config-identity-test-resources)` is needed any longer as well and if not remove it also.

Reply:
Does referencing `IDENTITY_TENANT_ID` directly in `LiveManagedIdentityTests.java` not work?

Reply:
We don't end up setting those variables so I had him switch to the ones we set for the azurepipelinecredential

Reply:
Also what about `sub-config-identity-test-resources` do you use anything from there? As we want to eliminate that as well eventually.

Reply:
There is AZURE_IDENTITY_MULTI_TENANT_CLIENT_SECRET as well.

Reply:
done.